### ISSUE TITLE
Auth help-functions for api

### ISSUE THREAD

**Issue by semsomi** [Fenixhack/alfarroba#2]
**[ this feature is depending on issue #1 ]**
In the api routes file there are auth checking functions like isAdmin or isOwnerOrAdmin.
let's create some more because we now have roles!

Look at isAdmin and implement the following:
**isLoggedIn**
copy isAdmin and change
`if (user && user.admin){`
to
`if (user){`

**isA** or  **hasUserRoleOf** or some better name
a function that either takes a string or an array describing what roles are accepted
ex:
`isA(['organizer', 'volunteer'])`

basically this function will in turn return another function that can be inserted in an express pipe,
ex psuedo code:

```
isA(roles){
  if(typeof roles === 'string'){
    roles = [roles]
  }
  const checkAuth = function(req, res, next){
    ---same as isAdmin, only check if the role is in the roles array---
  }
  return checkAuth
}

```
